src/Compilers/CSharp/Portable/Symbols/ConstraintsHelper.cs

```diff
@@ -1015,7 +1015,7 @@ static NullableFlowState getTypeArgumentState(in TypeWithAnnotations typeWithAnn
                     }
                     else
                     {
-                        result = (NullableFlowState)Math.Min((int)result.Value, (int)constraintState);
+                        result = result.Value.Meet(constraintState);
```

Comment:
Just to confirm: this is functionally equivalent, just a clean up, right?

Reply:
Yes, this is equivalent. This was follow up from an earlier PR.

---
In reply to: [466656042](https://github.com/dotnet/roslyn/pull/46405#discussion_r466656042) [](ancestors = 466656042)